cmd/kubeadm/app/preflight/checks.go

```diff
@@ -220,6 +246,7 @@ func RunInitMasterChecks(cfg *kubeadmapi.MasterConfiguration) error {
 		HostnameCheck{},
 		ServiceCheck{Service: "kubelet"},
 		ServiceCheck{Service: "docker"},
+		FirewalldCheck{ports: []int{int(cfg.API.BindPort), int(cfg.Discovery.BindPort), 10250}},
```

Comment:
I am quite unsure, I suspect we don't need etcd externally accessible, any idea if all the others are valid?